CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -243,16 +250,17 @@ exposed biology are affected by this even if a deployment method says they
 wouldn't. Use of biologic weapons are severely frowned upon in most civilized 
 regions of space.
 
-----Poison:	Lethal Toxin	-----
+==Poison:	Lethal Toxin ==
 Adrenaline Cost: 60
 Contagion Cost: 1
 Duration: 	almost instantaneous
 Range:		Payload (use in combination with other powers)
 Effect:		A biologic target afflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite 
```

Comment:
I missed mentioning another thing---

>prevents them from casting nanite 

Should probably say something like "prevent them from _using_ Nanite"...
Since we don't 'cast ablities/spells' anymore.